homeassistant/components/alexa/capabilities.py

```diff
@@ -953,6 +954,10 @@ def get_property(self, name):
             ):
                 return f"{cover.ATTR_POSITION}.{mode}"
 
+        if self.instance == f"{light.DOMAIN}.effect":
+            mode = self.entity.attributes.get(light.ATTR_EFFECT)
+            return f"{mode}"
```

Comment:
```suggestion
            return f"{light.ATTR_EFFECT}.{mode}"
```
Need to include identifier of the mode, in addition to the mode.